src/features/wallet/root/ActivityDetails/Payment.tsx

```diff
@@ -21,14 +22,23 @@ const Payment = ({ item, address }: Props) => {
     <Box flex={1}>
       <PaymentItem text={payer} mode="from" isMyAccount={payer === address} />
       {payments.map((p, index) => (
-        <PaymentItem
-          key={p.payee}
-          text={p.payee}
-          isMyAccount={p.payee === address}
-          mode="to"
-          isFirst={false}
-          isLast={index === payments.length - 1}
-        />
+        <Box key={p.payee}>
```

Comment:
Will `payee` always be unique?

Reply:
I think so, but let me double check you cant have the same payee multiple times.

Reply:
The blockchain doesn't allow it https://github.com/helium/blockchain-core/blob/master/test/blockchain_payment_v2_SUITE.erl#L196-L219